flang/include/flang/Optimizer/Builder/IntrinsicCall.h

```diff
@@ -185,6 +185,7 @@ struct IntrinsicLibrary {
   mlir::Value genAnint(mlir::Type, llvm::ArrayRef<mlir::Value>);
   fir::ExtendedValue genAny(mlir::Type, llvm::ArrayRef<fir::ExtendedValue>);
   mlir::Value genAtanpi(mlir::Type, llvm::ArrayRef<mlir::Value>);
+  mlir::Value genAtomAdd(mlir::Type, llvm::ArrayRef<mlir::Value>);
```

Comment:
Is there a reason to truncate Atomic to Atom?

Reply:
The formatting. It will make each line become 3 lines. I found it a lot less readable.

Reply:
Looks like it fits here. Are you talking about about the implementation or in the table?

Reply:
As someone that spending a lot of time grepping, I much rather having the full "atomic" spelling at the cost of extra lines in the table.